Tests/SonarScanner.MSBuild.Common.Test/RegularExpressions/WildcardPatternMatcherTest.cs

```diff
@@ -28,7 +28,78 @@ public class WildcardPatternMatcherTest
     /// <summary>
     /// Based on https://github.com/SonarSource/sonar-plugin-api/blob/master/plugin-api/src/test/java/org/sonar/api/utils/WildcardPatternTest.java.
     /// </summary>
-    [TestCategory(TestCategories.NoUnixNeedsReview)]
+    [TestCategory(TestCategories.NoLinux)]
+    [TestCategory(TestCategories.NoMacOS)]
+    [DataTestMethod]
+    [DataRow("Foo", "Foo", true)]
+    [DataRow("foo", "FOO", false)]
+    [DataRow("Foo", "Foot", false)]
+    [DataRow("Foo", "Bar", false)]
+    [DataRow("org/T?st.cs", "org\\Test.cs", true)]
```

Comment:
NIT. Can you turn all these into verbatim strings?
```suggestion
    [DataRow("org/T?st.cs", @"org\Test.cs", true)]
```